game/scripts/npc/abilities/lone_druid_spirit_bear.txt

```diff
@@ -54,7 +54,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "backlash_damage"                                 "20.0" //OAA
+        "backlash_damage"                                 "90.0" //OAA
```

Comment:
Just remove aghs effect, can keep the cd on aghs

Reply:
I said this multiple times: no